src/main/kotlin/racingCar/Car.kt

```diff
@@ -0,0 +1,15 @@
+package racingCar
+
+interface ICar {
+    var position: Int
+    fun moveFor(tryCount: Int)
+}
+class Car(private val Random: IRandom): ICar {
+    override var position: Int = 0
+
+    override fun moveFor(tryCount: Int) {
+        for (i: Int in 1..tryCount) {
```

Comment:
```suggestion
        for (i in 1..tryCount) {
```
타입 지정을 해주지 않아도 될 것 같습니다 ″̮